recipes/gklib/meta.yaml

```
@@ -23,12 +24,12 @@ requirements:
 test:
   commands:
 
-    - test -f ${PREFIX}/bin/cmpnbrs  # [unix]
-    - test -f ${PREFIX}/bin/csrcnv  # [unix]
-    - test -f ${PREFIX}/bin/fis  # [unix]
-    - test -f ${PREFIX}/bin/gkgraph  # [unix]
-    - test -f ${PREFIX}/bin/gkrw  # [unix]
-    - test -f ${PREFIX}/bin/m2mnbrs  # [unix]
+    - ${PREFIX}/bin/cmpnbrs --help  # [unix]
+    - ${PREFIX}/bin/csrcnv --help  # [unix]
+    - ${PREFIX}/bin/fis --help  # [unix]
+    - ${PREFIX}/bin/gkgraph --help  # [unix]
+    - ${PREFIX}/bin/gkrw --help  # [unix]
+    - ${PREFIX}/bin/m2mnbrs --help  # [unix]
```

They should be on PATH:
```suggestion
    - cmpnbrs --help  # [unix]
    - csrcnv --help  # [unix]
    - fis --help  # [unix]
    - gkgraph --help  # [unix]
    - gkrw --help  # [unix]
    - m2mnbrs --help  # [unix]
```

---

Oops thanks! Fixed by 38bf7b6